packages/common/test/customChains.spec.ts

```diff
@@ -98,6 +98,52 @@ describe('[Common]: Custom chains', () => {
     }
   })
 
```

Comment:
Did you make any substantial changes in this new test other than change the custom chain reference on line 106 to `PolygonAmoy`?  I don't see anything obvious so not sure we need a duplicate of the test above with a different chain name.